.travis.yml

```diff
@@ -4,25 +4,19 @@ python:
   - '2.7'
   - '3.6'
 install:
-  - sudo apt-get update
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
-  - conda info -a
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION pytorch scipy -c pytorch
-  - source activate test-environment
-  - pip install -U --process-dependency-links -e .[develop]
-  - pip install --upgrade pytest
+- sudo apt-get update
+- pip install --upgrade pip
+- pip install -U -q -e .[develop]
+# numpy upgrade needed for Python 2.7 only
+- pip install --upgrade pytest numpy
```

Comment:
what version of numpy is needed? Can we encode this in `setup.py`?

Reply:
Will check and try to fix this there.

Reply:
@lukasheinrich Fixed in commit a7d3c3a